doc/api/tls.markdown

```diff
@@ -615,36 +620,37 @@ Creates a new client connection to the given `port` and `host` (old API) or
 
   - `NPNProtocols`: An array of strings or `Buffer`s containing supported NPN
     protocols. `Buffer`s should have the following format:
-    `0x05hello0x05world`, where first byte is next protocol name's
-    length. (Passing array should usually be much simpler:
-    `['hello', 'world']`.)
+    `0x05hello0x05world`, where the first byte is the next protocol name's
+    length. (Passing an array is usually be much simpler: `['hello', 'world']`.)
 
-  - `ALPNProtocols`: An array of strings or `Buffer`s containing
+  - `ALPNProtocols`: An array of strings or `Buffer`s containing the
     supported ALPN protocols. `Buffer`s should have following format:
     `0x05hello0x05world`, where the first byte is the next protocol
-    name's length. (Passing array should usually be much simpler:
+    name's length. (Passing an array is usually be much simpler:
     `['hello', 'world']`.)
 
-  - `servername`: Servername for SNI (Server Name Indication) TLS extension.
+  - `servername`: Server name for the SNI (Server Name Indication) TLS 
+    extension.
 
   - `checkServerIdentity(servername, cert)`: Provide an override for checking
-    server's hostname against the certificate. Should return an error if verification
-    fails. Return `undefined` if passing.
+    the server's hostname against the certificate. Should return an error if 
+    verification fails. Returns `undefined` if passing.
 
-  - `secureProtocol`: The SSL method to use, e.g. `SSLv3_method` to force
-    SSL version 3. The possible values depend on your installation of
-    OpenSSL and are defined in the constant [SSL_METHODS][].
+  - `secureProtocol`: The SSL method to use, e.g., `SSLv3_method` to force
+    SSL version 3. The possible values depend on the version of OpenSSL 
+    installed in the environment and are defined in the constant 
+    [SSL_METHODS][].
 
   - `secureContext`: An optional TLS context object from
-     `tls.createSecureContext( ... )`. It could be used for caching client
-     certificates, key, and CA certificates.
+    `tls.createSecureContext( ... )`. It can be used for caching client
+    certificates, key, and CA certificates.
```

Comment:
Since the two others are multiples, maybe `client ceritificates, keys, and CA certificates` - also, do we consistently use `,` before `and` in enumerations? I think we don't so it's probably a good idea to remove the comma here.